multi_adv_ix_test.py

```diff
@@ -0,0 +1,36 @@
+import sparse
```

Comment:
Can we remove this file?

Reply:
Yes sorry, I removed them in latest commit. Those were there just for local testing and I had forgot to remove them.

Reply:
No need to apologize. 😄 I tend to be a bit too quick in mt reviews.